security/providers/oidc-common/src/main/java/io/helidon/security/providers/oidc/common/OidcConfig.java

```diff
@@ -1061,7 +1102,7 @@ public Builder scopeAudience(String audience) {
          * @param sameSite SameSite cookie attribute value
          * @return updated builder instance
          */
-        private Builder cookieSameSite(String sameSite) {
+        public Builder cookieSameSite(String sameSite) {
```

Comment:
Why was this method changed from private to public?

Reply:
This was a mistake in the first place - the methods should have been public, as they allow configuration that can be done using Config.